### PR TITLE
Visualize chromothripsis results with breakpoint data

### PR DESCRIPTION
**Pull request by LauraEgolf**
#### What scientific question is your analysis addressing?
Visualize the chromothripsis results (number of regions per sample) alongside breakpoint data from `analyses/chromosomal-instability/`. Mostly for the purpose of confirming that these results make sense (they should correlate).

#### What was your approach?
Created three simple plots (`plots/04-breakpoint-data`)

#### What GitHub issue does your pull request address?
#1007 (building off of #393)

#### Which areas should receive a particularly close look?
N/A, open to any suggestions.

#### Is there anything that you want to discuss further?
N/A

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes but we can do a thorough review once I have added all visualizations. I will submit a final PR to plot some of the candidate chromothripsis regions. (May also add other analyses/plots, to be discussed)

#### What types of results are included (e.g., table, figure)?
3 figures

#### What is your summary of the results?
As expected, the number of chromothripsis calls roughly correlates with the breakpoint counts calculated from both SV and CNV data.

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by LauraEgolf** [comment]
Sorry for the delay in getting this PR in - I'll submit the next one ASAP so that we can discuss results and additional analyses.

File: analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+chromoth_per_sample <- read_tsv(file.path(analysis_dir, "results", "chromothripsis_summary_per_sample.txt"))
+
+# Read in CNV and SV breakpoint data
+cnv_densities <- read_tsv(file.path(breakpoint_data_dir, "cnv_breaks_densities.tsv"))
+sv_densities <- read_tsv(file.path(breakpoint_data_dir, "sv_breaks_densities.tsv"))
+
+# Rename columns before merging
+cnv_densities <- cnv_densities %>%
+  rename(Kids_First_Biospecimen_ID = samples) %>%
+  rename(cnv_breaks_count = breaks_count)
```
Comment (jaclyn-taroni) [inline analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd:49]: I believe you could combine these `rename()` steps. Not particularly critical, though!
```suggestion
cnv_densities <- cnv_densities %>%
  rename(Kids_First_Biospecimen_ID = samples,
         cnv_breaks_count = breaks_count)
```

**Review by jaclyn-taroni — APPROVED** [review]
Thank you for submitting this @LauraEgolf and apologies for the delay in getting this reviewed. This looks good to me as is. I had one very minor comment about combining some steps, but I don't think that precludes getting this merged - more for your information!

File: analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd (hunk 1 above)
Comment (LauraEgolf) [reply analyses/chromothripsis/04-plot-chromothripsis-and-breakpoint-data.Rmd]: Looks good to me!

**Comment by LauraEgolf** [comment]
Thanks for the review! I'll be on vacation next week so I'll try to submit my final(ish) PR before I leave.

**Pull request merged**